src/mocks/assessmentAPI.ts

```diff
@@ -208,10 +208,33 @@ What's your favourite dinner food?
     comment: null,
     id: 0,
     library: mockSoundLibrary,
-    prepend: '',
-    postpend: '',
-    testcases: [],
-    solutionTemplate: '0th question mock solution template',
+    prepend: `// This is a mock Prepend!
+function fibonacci(n) {
+  if (n <= 2) {
+    return 1;
+  } else {
+    return fibonacci(n-1) + fibonacci(n-2);
+  }
+}`,
+    postpend: "// This is a mock Postpend! You shouldn't be able to see me!",
```

Comment:
If we have postpend and prepend, why are we only showing prepend on the UI? Or is postpend being shown also?

Reply:
The postpend is to allow mission authors to set up helper functions for grading purposes, so it is not displayed for users, but needed for grading purposes.

Reply:
Okay understood 👍